src/makeCalculator.js

```diff
@@ -5,6 +5,28 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+  };
+
+  calculator.reset = () => {
+    calculator.result = 0;
+
+    return calculator;
+  };
+
+  calculator.add = (number) => (calculator.result += number);
+  calculator.subtract = (number) => (calculator.result -= number);
+  calculator.multiply = (number) => (calculator.result *= number);
+  calculator.divide = (number) => (calculator.result /= number);
+
+  calculator.operate = (callback, number) => {
```

Comment:
The `operate` method should validate that `callback` is a function before calling it. Consider adding a check to ensure `callback` is a function to avoid potential runtime errors.